.pre-commit-config.yaml

```diff
@@ -0,0 +1,20 @@
+---
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.2.0
+    hooks:
+      - id: check-json
+      - id: check-yaml
+      - id: end-of-file-fixer
+      - id: trailing-whitespace
+
+  - repo: https://github.com/pre-commit/mirrors-prettier
+    rev: v2.6.2
+    hooks:
+      - id: prettier
+        files: \.(js|ts|jsx|tsx|css|less|html|json|markdown|md|yaml|yml)$
```

Comment:
I assume `pretty-format-json` is not included because prettier does its job, right?

Reply:
I will add the specific one and remove the extension from prettier

Reply:
I just want to remark that this one was optional (that is why it is not included in the list of requested changes). I wanted to understand the difference, but I see now that you have changed it that `pretty-format-json` does additional stuff, such as sorting the keys alphabetically. But that is uncritical and also prettier could be used.